data/json/professions.json

```
@@ -620,6 +620,25 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "baseball_player",
+    "name": "Baseball Player",
+    "description": "You were playing at the local team.  With no one left to play baseball, you can eventually use your best friend for another purpose.  Home run!",
+    "points": 1,
+    "skills": [
+      { "level": 3, "name": "throw" },
+      { "level": 1, "name": "melee" }
+    ],
```

```suggestion
    "skills": [ { "level": 3, "name": "throw" }, { "level": 1, "name": "melee" } ],
```
Arrays are usually formatted like this.

---

Thanks! Since its my frst time contributing something, I knew I am going to make lots of mistakes.

---

No problem! This isn't even really a mistake, we just like to keep the files formatted a certain way.
You can go ahead and press "commit suggestion" on this thread to automatically commit it onto your branch, or "add to batch" to put both changes in a single commit.